src/containers/App/reducer.js

```diff
@@ -0,0 +1,18 @@
+import type {AppState} from './types';
+
+const initialState: AppState = {
+  buildInformation: {
+    frontEndBuldNumber: process.env.REACT_APP_BUILD_NUMBER,
+    deploymentEnvironment: process.env.REACT_APP_DEPLOYMENT_ENV,
+  },
+};
+
+const app = (state: AppState = initialState, action): AppState => {
+  const {type} = action;
+  switch (type) {
```

Comment:
What's the purpose of this switch?

Reply:
switch is for handling each type of action that is dispatched and handled by the reducer to make changes to the state.

Reply:
It currently does nothing other than the default. Is the switch here only as a placeholder because we are 100% certain more options will be here later?

Reply:
100% sure but either way you still need the default 100% of the time, this way you return the initial state of the application. Heck redux will throw a warning if the reducer switch statement doesn't not contains the default case returning the state.

Reply:
I'm not questioning whether or not to return the default value. I'm questioning the switch.